doc/classes/Projection.xml

```diff
@@ -81,12 +93,15 @@
 			<param index="4" name="z_far" type="float" />
 			<param index="5" name="flip_fov" type="bool" default="false" />
 			<description>
+				Creates a new [Projection] that projects positions in a frustum with the given size, X:Y aspect ratio, offset, and clipping planes.
+				[param flip_fov] determines whether the projection's field of view is flipped over its diagonal.
 			</description>
 		</method>
 		<method name="create_light_atlas_rect" qualifiers="static">
 			<return type="Projection" />
 			<param index="0" name="rect" type="Rect2" />
 			<description>
+				Creates a new [Projection] that projects positions into the given [Rect2].
```

Comment:
Similar to ``create_fit_aabb`` this is used in a very specific part of the rendering pipeline and not be broadly useful. But the description is accurate for what it does